sources/api/migration/migrations/v1.20.0/static-pods-services-cfg-v0-1-0/src/main.rs

```diff
@@ -0,0 +1,23 @@
+use migration_helpers::common_migrations::{ListReplacement, ReplaceListsMigration};
+use migration_helpers::{migrate, Result};
+use std::process;
+fn run() -> Result<()> {
+    migrate(ReplaceListsMigration(vec![
+        ListReplacement {
+            setting: "services.static-pods.configuration-files",
+            old_vals: &[],
+            new_vals: &[],
```

Comment:
isn't this missing the new list contents? surprised it seems to have worked fine on upgrade